app/http/routes/DeleteRouter.ts

```diff
@@ -0,0 +1,45 @@
+import express from "express";
+import slowDown from 'express-slow-down';
+import Arkitektonika, {SCHEMATIC_DIR} from "../../Arkitektonika";
+import path from "path";
+import * as fs from "fs";
+
+/**
+ * Configures the rate limiter for the delete-route because brute force is kind of not cool
+ * Allows 30 requests per minute = 1 request every 2 seconds.
+ * Each request above that limit will load for 500ms extra.
+ * The 31st request will take additional 500ms to fulfill, 32nd additional 1000ms, ...
+ */
+const LIMITER = slowDown({
+    windowMs: 1000 * 60,
+    delayAfter: 30,
+    delayMs: 500
+});
```

Comment:
Considering people can selfhost this interface, the default may be fine for our needs, but should be configurable for standalone instances.